uppsrc/CtrlCore/X11Proc.cpp

```diff
@@ -418,17 +419,20 @@ void Ctrl::EventProc(XWindow& w, XEvent *event)
 		}
 		break;
 	case MotionNotify:
-		while(XCheckWindowEvent(Xdisplay, top->window, PointerMotionMask, event));
-		EndIgnore();
-		mousePos = Point(event->xmotion.x_root, event->xmotion.y_root);
-		Xeventtime = event->xmotion.time;
-		Point p = mousePos - Xbuttonpos;
-		if(max(abs(p.x), abs(p.y)) > 4)
-			Xbuttontime = Xeventtime - 0x80000000;
-		sModState = event->xmotion.state;
-		DispatchMouse(MOUSEMOVE, Point(event->xmotion.x, event->xmotion.y));
-		DoCursorShape();
-		break;
+		Top *top = GetTop();
```

Comment:
This line should be removed:
if (top) // Global Ctrl variable... {
    while(XCheckWindowEvent(Xdisplay, utop->window, PointerMotionMask, event));
    ....